src/three/math/Ellipsoid.d.ts

```diff
@@ -34,4 +35,12 @@ export class Ellipsoid {
 	getEastNorthUpFrame( lat: number, lon: number, target: Matrix4 ): Matrix4;
 	getEastNorthUpAxes( lat: number, lon: number, vecEast: Vector3, vecNorth: Vector3, vecUp: Vector3, point?: Vector3 ): void;
 
+	intersectRay( ray: Ray, target: Vector3 ): Vector3 | null;
+
+	calculateHorizonDistance( latitude: number, elevation: number ): number;
+	calculateEffectiveRadius( latitude: number ): number;
+
```

Comment:
These functions were added for GlobeControls and I don't want to consider them public at the moment. If they're going to be made public I'd like to consider some more descriptive names for the functions.

Reply:
I am using `calculateEffectiveRadius` in trigonometry calculations (linear distance between two GPS coordinates) for an animation of globe rotation to know if the place the camera is currently looking at is within a certain tolerance radius of the animation destination. If it is in the radius, the camera doesn't move; if it is outside, the camera animates.